perun-core/src/test/java/cz/metacentrum/perun/core/entry/ServicesManagerEntryIntegrationTest.java

```diff
@@ -1187,6 +1190,59 @@ public void blockAndUnblockServiceOnDestinations() throws Exception {
 		assertFalse("Service should NOT be blocked on the richDestination3", perun.getServicesManager().isServiceBlockedOnDestination(sess, service2, destination3.getId()));
 	}
 
+	@Test
+	public void blockAndUnblockServicesOnFacility() throws Exception {
+		System.out.println(CLASS_NAME + "blockAndUnblockServicesOnFacility");
```

Comment:
Wouldn't it be better to split the test for block and unblock ?

Reply:
There are more cases when we test some action and its opposite in the same test since most of the code would be duplicated. But if it makes more sense to you, I can split it with no problem.

Reply:
In that case it is fine with me.